plugins/SitesManager/Controller.php

```diff
@@ -176,12 +176,20 @@ public function siteWithoutData()
         $emailTemplateData['cms'] = $this->siteContentDetector->cms;
 
         $emailContent = $this->renderTemplateAs('@SitesManager/_trackingCodeEmail', $emailTemplateData, $viewType = 'basic');
+        $descriptionLine1AdditionalLinks = '';
+        Piwik::postEvent('SitesManager.descriptionLine1AdditionalLinks', [&$descriptionLine1AdditionalLinks]);
+
+        if (Piwik::hasUserSuperUserAccess()) {
+            $inviteUserLink = $this->getInviteUserLink();
+            $descriptionLine1AdditionalLinks = (!empty($descriptionLine1AdditionalLinks) ? ', ': ' or ') . Piwik::translate('SitesManager_SitesWithoutDataInviteAnotherUser', ['<a target="_blank" rel="noreferrer noopener" href="'.$inviteUserLink.'">', '</a>']) . $descriptionLine1AdditionalLinks;
```

Comment:
Here the `or` would actually be untranslated.

Reply:
removed this code